presto-geospatial/src/main/java/com/facebook/presto/geospatial/BingTileFunctions.java

```diff
@@ -206,8 +206,8 @@ private static BingTile latitudeLongitudeToTile(double latitude, double longitud
         double y = 0.5 - Math.log((1 + sinLatitude) / (1 - sinLatitude)) / (4 * Math.PI);
 
         int mapSize = mapSize(zoomLevel);
-        int tileX = (int) clip(x * mapSize + 0.5, 0, mapSize - 1);
```

Comment:
My understanding is that if `x*mapSize` is an integer, then `x*mapSize + 0.5` will be around `n.5`, which might be round to `n` or `n+1` due to the float precision error? :)